lib/supa/commands/attributes.rb

```diff
@@ -0,0 +1,23 @@
+module Supa
+  module Commands
+    class Attributes < Supa::Command
+      def represent
+        names.each do |name|
+          Supa::Commands::Attribute.new(
+            @subject, representer: @representer, tree: @tree, name: name, options: attribute_options
```

Comment:
@Strech It will allow to use all `attribute` command options. Some of the don't make sense to use when using `attributes` command.